log4j-core/src/main/java/org/apache/logging/log4j/core/config/builder/api/package-info.java

```diff
@@ -20,7 +20,7 @@
  * @since 2.4
  */
 @Export
-@Version("2.20.1")
+@Version("2.21.0")
```

Comment:
Always target the version this feature will (supposedly) be shipped with:

```suggestion
@Version("2.25.0")
```

Reply:
Done